client/src/styles/main.css

```diff
@@ -257,4 +261,32 @@ a:link {
 .icons--display {
   padding-bottom: 1.5rem;
   padding-top: 1.5rem;
+}
+
+.tile--display {
+  display: flex;
+  padding-right: 2rem;
+}
+
+.tile {
+  width: 20%
+}
+
+tile__inside {
```

Comment:
Any reason why you switch between using underscore in this naming and dash in other places? Consider maintaining a consistent pattern if there is no strong reason.